src/base/QXmppOmemoData.cpp

```diff
@@ -695,7 +695,7 @@ void QXmppOmemoElement::setPayload(const QByteArray &payload)
 ///
 /// \return the found OMEMO envelope
 ///
-std::optional<QXmppOmemoEnvelope> QXmppOmemoElement::searchEnvelope(const QString &recipientJid, uint32_t recipientDeviceId) const
+std::optional<QXmppOmemoEnvelope> QXmppOmemoElement::searchEnvelope(const QString &recipientJid, const uint32_t recipientDeviceId) const
```

Comment:
I know nobody else than you doing this, the general coding style in QXmpp is without this const when parameters are passed by value.

Reply:
See 75d27e8093d900044c305a0cdfbed3e32784028c